src/main/scala/snabbdom/hFunction.scala

```diff
@@ -84,41 +67,105 @@ object DataObject {
   type StyleValue = String | js.Dictionary[String]
   type KeyValue = String | Double  // https://github.com/snabbdom/snabbdom#key--string--number
 
-  def apply(attrs: js.Dictionary[AttrValue],
-            on: js.Dictionary[js.Function1[Event, Unit]],
-            hooks : Hooks = Hooks()
-           ): DataObject = apply(attrs, js.Dictionary.empty, js.Dictionary.empty, on, hooks, js.undefined)
-
-
-  def apply(attrs: js.Dictionary[AttrValue],
-            props: js.Dictionary[PropValue],
-            style: js.Dictionary[StyleValue],
-            on: js.Dictionary[js.Function1[Event, Unit]],
-            hook: Hooks,
-            key: js.UndefOr[KeyValue]
-           ): DataObject = {
-
-    val _attrs = attrs
-    val _props = props
-    val _style = style
-    val _on = on
-    val _hook = hook
-    val _key = key
-
-    new DataObject {
-      val attrs: js.Dictionary[AttrValue] = _attrs
-      val props: js.Dictionary[PropValue] = _props
-      val style: js.Dictionary[StyleValue] = _style
-      val on: js.Dictionary[js.Function1[Event, Unit]] = _on
-      val hook: Hooks = _hook
-      val key: UndefOr[KeyValue] = _key
+  def empty: DataObject = new DataObject { }
+}
+
+// @js.native
```

Comment:
Why are these comments here?

Reply:
I think we should either remove them or write a short note, why we left them here.

```suggestion
// These are the original facades for snabbdom thunk. But we implement our own, so that for equality checks, the equals method is used.
// @js.native
```